### PR TITLE
Use new use_max_points_column arguments to turn on feature in PyLmod

### LINKED ISSUE #36: Transmit raw score and max score to remote gradebook
Currently lmod_proxy sends calculated scores to the remote gradebook.  The requested change is to send the raw score and the max score.

To send the max score, the assignment must be created by lmod_proxy.  (In the existing implementation, assignments are created in the body of the PyLmod call that sends scores to the remote gradebook.)

### PR DESCRIPTION
**Pull request by noisecapella**
#### What are the relevant tickets?
Fixes #36

#### What's this PR do?
 - Points to a new version of pylmod
 - Enables the max points column feature by adding a new argument to `spreadsheet2gradebook`

#### Where should the reviewer start?
lmod_proxy/edx_grades/actions.py

#### How should this be manually tested?
TODO

### CODE REVIEW

File: lmod_proxy/edx_grades/actions.py [hunk 1]
```diff
@@ -31,7 +31,10 @@ def post_grades(gradebook, form):
     results = None
     try:
         results, time_taken = gradebook.spreadsheet2gradebook(
-            csv_file=csv_file, approve_grades=approve_grades
+            csv_file=csv_file, approve_grades=approve_grades,
```
Comment (giocalitri) [inline lmod_proxy/edx_grades/actions.py]: nit: one arg per line

File: lmod_proxy/tests/test_edx_grades.py [hunk 2]
```diff
@@ -320,6 +320,34 @@ def test_post_grades_approve(self):
         )
         self.assertEqual(data, [])
 
+    def test_max_points(self):
+        """Validate max points arguments"""
+        from lmod_proxy.edx_grades.forms import EdXGradesForm
+        from lmod_proxy.edx_grades.actions import post_grades
```
Comment (giocalitri) [inline lmod_proxy/tests/test_edx_grades.py]: question: why the imports are here?

File: lmod_proxy/tests/test_edx_grades.py [hunk 3]
```diff
@@ -320,6 +320,34 @@ def test_post_grades_approve(self):
...
+        gradebook = mock.MagicMock()
+        gradebook_return = {'data': {'test': 'foo'}}
+        gradebook.spreadsheet2gradebook.return_value = (gradebook_return, 1)
+
+        with self.app.app_context():
+            with mock.patch.dict(
+                'lmod_proxy.edx_grades.actions.current_app.config',
+                {'LMODP_APPROVE_GRADES': 'a'}
+            ):
+                _, data, _ = post_grades(gradebook, form)
```
Comment (giocalitri) [inline lmod_proxy/tests/test_edx_grades.py]: why not using just one context manager?

**Comment by giocalitri** [comment]
the tests are not passing

**Comment by noisecapella** [comment]
The tests require pylmod==0.2.0 which doesn't exist yet

File: lmod_proxy/tests/test_edx_grades.py (hunk 2 above)
Comment (noisecapella) [inline lmod_proxy/tests/test_edx_grades.py]: I adapted an existing test that used imports like this. I'll move them all to the top

**Comment by giocalitri** [comment]
ok: what about the other comments?

**Comment by noisecapella** [comment]
I think I addressed them all

File: lmod_proxy/edx_grades/actions.py (hunk 1 above)
Comment (giocalitri) [inline lmod_proxy/edx_grades/actions.py]: this one is the only one left

**Comment by noisecapella** [comment]
I addressed your comment

**Comment by giocalitri** [comment]
The code is fine :+1: 

someone should test functionality

**Comment by pdpinch** [comment]
That would be @pwilkins as well.

**Comment by bdero** [comment]
I can help verify the functionality of this once PyLmod is released, which will happen after this PR is merged: https://github.com/mitodl/PyLmod/pull/61

**Comment by bdero** [comment]
pylmod is now released and I've pushed the package to pypi. I'm restarting the build for this PR so we can verify it's working now.

**Comment by bdero** [comment]
@noisecapella It looks like there's just a small pep8 failure. After that, this should be good to merge. I've verified the functionality a bunch of times over the past couple of days.

**Comment by bdero** [comment]
And now it's passing, just giving it one more manual test with the new pylmod package for good measure

**Pull request merged**